src/main/js/portal/main/components/preview/PreviewTimeSerie.tsx

```diff
@@ -165,6 +165,10 @@ export default class PreviewTimeSerie extends Component<OurProps, OurState> {
 		const {xAxis, yAxis, y2Axis, type} = getAxes(options, preview, specSettings);
 		const objIds = preview.items.map((i: CartItem) => getLastSegmentInUrl(i.dobj)).join();
 
+		const excludeChartType = items.reduce((acc: boolean, curr: PreviewItem) => {
+			return acc || previewExcludeChartType.specs.includes(curr.spec);
+		}, false);
+
```

Comment:
I don't think it's necessary to go through all the items as they need to contain the same dataset to be displayed together.